engine/lib/admin.php

```diff
@@ -226,11 +226,8 @@ function _elgg_admin_init() {
 	elgg_register_action('admin/site/set_robots', '', 'admin');
 	elgg_register_action('admin/site/set_maintenance_mode', '', 'admin');
 
-	elgg_register_action('admin/upgrades/upgrade_comments', '', 'admin');
-	elgg_register_action('admin/upgrades/upgrade_datadirs', '', 'admin');
-	elgg_register_action('admin/upgrades/upgrade_discussion_replies', '', 'admin');
-	elgg_register_action('admin/upgrades/upgrade_comments_access', '', 'admin');
```

Comment:
The new way doesn't require every upgrade to have their own action and view like in Elgg 2.